spk/shairport-sync/src/installer.sh

```diff
@@ -59,6 +61,11 @@ preupgrade ()
 {
     # Stop the package
     ${SSS} stop > /dev/null
+    
+    # Remove firewall config
```

Comment:
Postinst is run during a package upgrade as well (check Synology's dev guide), so no need for that. Add/update rules during `postinst`, and remove them during `preuninst` (with a check during `preuninst`, just as you have now)